### DISCUSSION TITLE
Good afternoon)When will you be able to add the aerospike plugin?

### DISCUSSION THREAD

**Discussion by vovakaplenko** [trinodb/trino#16410] [Q&A] [answered]
To be able to connect and it will load itself!

**Comment by hashhar** [comment, marked as answer]
Have you seen https://download.aerospike.com/enterprise/download/connectors/aerospike-trino/notes.html?

We cannot distributed it with Trino itself however due to license incompatibilities and because it seems to require some license/feature key from Aerospike to use it.

**Comment by hashhar** [comment]
Feel free to join our [Slack](https://trino.io/slack.html) and continue the conversation there.

**Comment by vovakaplenko** [comment]
You do not understand I have a license for aerospike! I need that you had a plugin to connect to it.https://github.com/trinodb/trino/tree/master/plugin there should be

**Comment by hashhar** [comment]
Yes, you can download the plugin from Aerospike themselves from https://download.aerospike.com/enterprise/download/connectors/aerospike-trino/notes.html.

There are also instructions on how to add it to a cluster at https://docs.aerospike.com/connect/presto/installing-and-configuring-presto-connector

**Reply by vovakaplenko** [reply]
I need to install trinodb in k8s,connector aerospike ! How do I do it ?

**Comment by vovakaplenko** [comment]
I found a repository where this is done on the docker https://github.com/aerospike/trino-aerospike.docker
